docs/version0.46.md

```diff
@@ -37,6 +37,10 @@ The following new features and notable changes since version 0.45.0 are included
 
 Eclipse OpenJ9&trade; release 0.46.0 supports OpenJDK 8, 11, 17, 21, and 22.
 
+Support for running OpenJ9 with all versions of OpenJDK on CentOS 6.10, CentOS 7.9, Red Hat Enterprise Linux (RHEL) 6.10, and RHEL 7.8 is removed.
```

Comment:
I think this should be more generic so there isn't a misunderstanding. Rather than 6.10 use 6 and rather than 7.9 or 7.8 use 7.